packages/i18n/src/locales/en.i18n.json

```diff
@@ -1125,8 +1126,8 @@
   "Common_Access": "Common Access",
   "Commit": "Commit",
   "Community": "Community",
-  "Contextualbar_resizable": "Contextual bar resizable",
-  "Contextualbar_resizable_description": "Allows you to adjust the size of the contextual bar by simply dragging, giving you instant customization and flexibility",
+  "Contextualbar_resizable": "Resizable contextual bar",
```

Comment:
I know probably someone asked you to improve those translations, but keep in mind this could be improved in a different pr, its not 100% related with the feature itself